### ISSUE TITLE
Footer config locked

### ISSUE THREAD

**Issue by geopast** [sharetribe/sharetribe#3740]
_Please add only one problem per Issue. If you are facing multiple problems, add multiple Issues._

##### Sharetribe Version: 7.6.0

_Share the git commit hash of the version of Sharetribe that you are using. You can find it by running `git rev-parse HEAD` in your Sharetribe directory._

##### Environment: a274e2122703cfc8de9add9ce8c214bd318c22a7

_What's your environment? Mac OS X, Heroku, or something else?_

Ubuntu 16.04.4 LTS

_What's your node version? Run the following commands and copy the result here:_

```
> node -v

v7.8.0

> npm -v
```
4.2.0

##### Server mode:

_Are you running in development or production mode?_

Production

##### Description:

_In our own words, briefly describe the issue. Please add enough details, yet be concise._

The admin settings section for footer displays a message only relevant to hosted Go - not the open source version.

##### Steps To Reproduce:

_List the actions that will reproduce the issue._

_This is the **most important** part of the issue report. If we're not able to reproduce the issue, there's not much we can do about it. Please provide precise steps how to reproduce the issue._

1. _Step 1_ Go to admin settings
1. _Step 2_ Click on Footer
1. _Step 3_ See message attached

Expected result: _What was the expected result after running the steps above?_

I should be able to edit the footer

Actual result: _What was the actual result after running the steps above?_

The footer is not active.
![Screen Shot 2019-04-30 at 9 12 39 pm](https://user-images.githubusercontent.com/10397157/56958579-f9fa4500-6b8d-11e9-8e19-db8237c61fca.png)

##### Additional information

_Please provide any additional information that helps debugging, e.g. log output or stack trace of the error_

**Comment by thomasmalbaux** [comment]
Have you ran the following via Rails command (replace with your `community_id` of course)?

`FeatureFlagService::API::Api.features.enable(community_id: YOUR_COMMUNITY_ID, features: [:footer])`

`PlanService::Store::Plan::PlanModel.create(community_id: YOUR_COMMUNITY_ID, status: "active", features: {"whitelabel"=>true, "admin_email"=>true, "footer"=>true}, expires_at: Time.current + 20.years)`

**Comment by geopast** [comment]
Thanks! That worked!

**Comment by webprobox** [comment]
I just updated my site to version 9.0.0 and tried to run Rails commands:

FeatureFlagService::API::Api.features.enable(community_id: 1, features: [:footer])

PlanService::Store::Plan::PlanModel.create(community_id: YOUR_COMMUNITY_ID, status: "active", features: {"whitelabel"=>true, "admin_email"=>true, "footer"=>true}, expires_at: Time.current + 20.years)

Database was updated, but admin still had footer option disabled.

Then I searched forum and modified footer_service.rb  plan_footer_disabled function function to return false and restarted server. 

This made footer option to become available for editing in admin, but it does not show on my website. I tried almost everything, but lack coding skills to find the issue. How to activate this footer?
